doc/modules/mod_event_pusher_sns.md

```diff
@@ -13,19 +13,97 @@ Full topics for notifications (ARN as defined in [Amazon Resource Names][aws-arn
 
 ### Options
 
-* **presence_updates_topic** (string, default: unset) - Defines Amazon SNS Topic for presence change notifications. Remove this option to disable these notifications.
-* **pm_messages_topic** (string, default: unset) - Defines Amazon SNS Topic for private message notifications. Remove this option to disable these notifications.
-* **muc_messages_topic** (string, default: unset) - Defines Amazon SNS Topic for group message notifications. Remove this option to disable these notifications.
-* **plugin_module** (atom, default: 'mod_event_pusher_sns_defaults') - Sets a callback module used for creating user's GUID used in notifications (from user's JID) and for defining custom attributes attached to a published SNS message.
-* **muc_host** (string, default: "conference.@HOST@") - Messages from this MUC host will be sent to the set SNS topic for MUCs.
-* **sns_host** (string, default: unset) - URL to the Amazon SNS service. The URL may be in [virtual host form][aws-virtual-host], and for AWS needs to point at a specific regional endpoint. The scheme, port and path specified in the URL will be used to publish notifications via HTTP POST method.
-* **region** (string, default: unset) - The [AWS region][aws-region] to use for requests.
-* **access_key_id** (string, default: unset) - [ID of the access key][aws-keys] to use for authorization.
-* **secret_access_key** (string, default: unset) - [Secret access key][aws-keys] to use for authorization.
-* **account_id** (string, default: unset) - 12 digit number as defined in [AWS Account Identifiers][aws-acct-identifier] to use for creating TopicArn for publishing notifications.
-* **pool_size** (integer, default: 100) - Worker pool size for publishing notifications
-* **publish_retry_count** (integer, default: 2) - Retry count in case of a publish error
-* **publish_retry_time_ms** (integer, default: 50) - Base exponential backoff time (in ms) for publish errors
+#### `modules.mod_event_pusher_sns.presence_updates_topic`
+* **Syntax:** string
+* **Default:** `""`
```

Comment:
```suggestion
* **Default:** no option
```

Reply:
What does this mean? In the validator it checks it is a string. If it's supposed to mean that there's no default, please check how no defaults are told in different modules, to keep it consistent.

Reply:
Anyway, just saying that `presence_updates_topic = ""` would probably crash module or do something strange.
It's also not validated.

Reply:
What do you mean by not validated? It is checked to pass `validate_string`. Should it check to pass a non-empty string validation?